services/QuillLMS/app/services/clever_integration/classroom_creator.rb

```diff
@@ -0,0 +1,35 @@
+module CleverIntegration
+  class ClassroomCreator
+    attr_reader :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def run
+      Classroom.create!(
+        clever_id: clever_id,
+        grade: grade,
+        name: name,
+        synced_name: synced_name
+      )
+    end
+
+
+    private def clever_id
```

Comment:
After reading this [post](https://www.codewithjason.com/dont-wrap-instance-variables-attr_reader-unless-necessary/) last week I was re-evaluating my use of `attr_reader` since it breaks encapsulation.  On second thought I will put them back in since it's easier to read and more consistent with previous code.